pyodk/endpoints/submissions.py

```diff
@@ -159,12 +153,165 @@ def get_table(
         except PyODKError as err:
             log.error(err, exc_info=True)
             raise err
-        else:
-            response = self.session.get_200_or_error(
-                url=self.urls.get_table.format(
-                    project_id=pid, form_id=fid, table_name=table
-                ),
-                logger=log,
-                params=params,
+
+        response = self.session.response_or_error(
+            method="GET",
+            url=self.urls.get_table.format(project_id=pid, form_id=fid, table_name=table),
+            logger=log,
+            params=params,
+        )
+        return response.json()
+
+    def post(
```

Comment:
In the spirit of matching informal/api docs language for naming library methods, how about `create`? This matches the header at https://odkcentral.docs.apiary.io/#reference/submissions/submissions/creating-a-submission When I had previously commented on `read`, I didn't want to deter from using all CRUD words! Especially now that we have the generic HTTP methods, I think "how do people talk about this thing" is a good guiding question.

Reply:
Renamed to "create"